.travis.yml

```diff
@@ -20,27 +25,20 @@ before_script:
     export PATH=$HOME/.local/bin:$PATH
 
 script:
-# TEMPORARY UNTIL SERDE 0.7 IS RELEASED
-- |
-  mkdir .travis-deps
-  mkdir .cargo
-  echo 'paths = ["./.travis-deps/serde"]' > .cargo/config
-  git clone https://github.com/serde-rs/serde .travis-deps/serde
-  pushd .travis-deps/serde
-  git reset --hard origin/master
-  popd
 - |
-    (cd json && travis-cargo build) &&
-    (cd json && travis-cargo test) &&
-    (cd json_tests && travis-cargo test -- --features with-syntex --no-default-features) &&
-    (cd json_tests && travis-cargo --only nightly test -- features nightly-testing) &&
-    (cd json_tests && travis-cargo --only nightly bench -- features nightly-testing) &&
-    (cd json_tests && travis-cargo --only nightly bench -- --features "nightly-testing with-syntex" --no-default-features) &&
-    (cd json && travis-cargo --only stable doc)
+    if [ -z $BENCH ]; then
```

Comment:
I don't epeak bash... What does the -z do, and why is the bench flag set to 1 on nightly?